src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -173,14 +181,47 @@ public void handleMessage(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter
 
     @Override
     public void reset() {
+        String d = serverContext.getDataStore().getLogDir();
+        localLog.close();
+        if (d != null) {
+            Path dir = FileSystems.getDefault().getPath(d);
+            String prefixes[] = new String[]{"log"};
+
+            for (String pfx : prefixes) {
```

Comment:
again, I prefer not to see explicit pathname manipulations like thie inside the main logic of LogUnitServer